includes/widgets/playing-cards.php

```diff
@@ -0,0 +1,267 @@
+					'8'  => '8',
+					'9'  => '9',
+					'10' => '10',
+					'J'  => esc_html_x( 'J', 'Playing cards value', 'elementor' ),
+					'Q'  => esc_html_x( 'Q', 'Playing cards value', 'elementor' ),
+					'K'  => esc_html_x( 'K', 'Playing cards value', 'elementor' ),
+					'A'  => esc_html_x( 'A', 'Playing cards value', 'elementor' )
+				],
+				'default'   => '2',
+				'separator' => 'before',
+			]
+		);
+
+		$this->add_control(
+			'cards_list',
+			[
+				'label' => esc_html__( 'Hand', 'elementor' ),
+				'type' => Controls_Manager::REPEATER,
+				'fields' => $repeater->get_controls(),
+				'default' => [],
+				'title_field' => '{{ card_type }} ({{ card_value }})',
+			]
+		);
+
+		$this->add_control(
+			'hover_animation',
+			[
+				'label' => esc_html__( 'Hover Animation', 'elementor' ),
+				'type' => Controls_Manager::SELECT,
+				'options' => [
+					'pulse' => esc_html__( 'Pulse', 'elementor' ),
+					'bounce' => esc_html__( 'Bounce', 'elementor' ),
+				],
+				'default' => 'pulse',
+				'separator' => 'before',
+			]
+		);
+
+		$this->end_controls_section();
+	}
+
+	/**
+	 * Render playing cards widget output on the frontend.
+	 *
+	 * Written in PHP and used to generate the final HTML.
+	 *
+	 * @since 3.12.1
+	 * @access protected
+	 */
+	protected function render() {
+		$cards = $this->get_settings_for_display( 'cards_list' );
+		$hover_animation = $this->get_settings_for_display( 'hover_animation' );
+
+		$types = [
+			'spade' => '♠',
+			'diamond' => '♦',
+			'heart' => '♥',
+			'club' => '♣',
+		];
+		?>
+		<div class="elementor-playing-cards elementor-playing-cards-<?php echo esc_attr( $hover_animation ); ?>">
+			<span class="elementor-screen-only" role="heading">
+				<?php echo esc_html__( 'Playing Cards', 'elementor' ); ?>
+			</span>
+
+			<?php
+			foreach ( $cards as $card ) {
+				$type = $card['card_type'];
+				$value = $card['card_value'];
+				?>
+				<div
+					class="elementor-playing-card elementor-playing-card-<?php echo esc_attr( $card['_id'] ); ?> elementor-playing-card-<?php echo esc_attr( $type ); ?>"
+					tabindex="0">
+					<span class="elementor-screen-only" role="heading">
+						<?php echo esc_html__( sprintf( 'Playing card of type %s and value %s', $type, $value ), 'elementor' ); ?>
+					</span>
+
+					<div class="elementor-playing-card-header"><?php echo esc_html( $value ); ?></div>
+					<div class="elementor-playing-card-body"><?php echo $types[ $type ] ?? ''; ?></div>
+					<div class="elementor-playing-card-footer"><?php echo esc_html( $value ); ?></div>
+				</div>
+			<?php } ?>
+		</div>
+		<?php
+	}
+
+	/**
+	 * Render playing cards widget output in the editor.
+	 *
+	 * Written as a Backbone JavaScript template and used to generate the live preview.
+	 *
+	 * @since 3.12.1
+	 * @access protected
+	 */
+	protected function content_template() {
+		?>
+		<# var getCardSymbol = function( type ) {
+			var types = {
+				spade: '♠',
+				diamond: '♦',
+				heart: '♥',
+				club: '♣',
```

Comment:
@pkniazevych Interesting, are these Unicode characters or what?

Reply:
Yeah, I find it better than adding our own assets or using emojis. Should be well supported.